### ISSUE TITLE
Bold or Italic incorrect adds new line in pdf

### ISSUE THREAD

**Issue by dannynicolas** [Aymkdn/html-to-pdfmake#10]
**Version:  v.1.0.6** 

It seems that using mutiple tags translates poorly to pdfmake. 

eg. if this data sent to html-to-pdfmake:
 
`<ul><li> Excepturi quos <strong>accusantium</strong> autem impedit. </li></ul>`

It gets printed in the pdf as three lines: 

- Excepturi quos
 **accusantium**
 autem impedit. 

It should be:
- Excepturi quos **accusantium** autem impedit.

**Comment by Aymkdn** [comment]
I've just left on vacation for two weeks. I'll have a look at it on my return.

**Comment by Aymkdn** [comment]
Mmhhh... Actually, how do you do it with pure PDFMake code? Because whatever I tried on http://pdfmake.org/playground.html it always has the same behavior...

So, could you share the PDFMake code you use?

**Comment by Aymkdn** [comment]
Nevermind, I finally found it : 
```
ul: [
      {text: [
         text: ' Excepturi quos ', marginLeft: 5 },
          { text: 'accusantium', bold: true },
          { text: ' autem impedit. ' }
     ]}
]
```

I'm working on it now.

**Comment by Aymkdn** [comment]
That should be resolved with v1.0.7
